test/test-cases/scale/saic/sai-api/test_sai_api_vnet_acl_in.py

```diff
@@ -0,0 +1,104 @@
+import json
+import time
+from pathlib import Path
+from pprint import pprint
+
+import pytest
+import saichallenger.dataplane.snappi.snappi_traffic_utils as stu
```

Comment:
If these test cases are only to test APIs and no traffic, let's remove all unused imports. Make the files as lean as possible.